index.html

```diff
@@ -81,12 +82,12 @@ <h1 class="header-title section-title">
         </div>
         <label>
           <input type="text" name="username"
-                 maxlength="25" required
+                 minlength="" maxlength="25" required
```

Comment:
```suggestion
                 minlength="2" maxlength="25" required
```
> имя персонажа не может содержать менее 2 символов;
ты написала правильное свойство, только забыла указать число